<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Main_QW">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="Main_QF">
      <property name="styleSheet">
       <string notr="true">QFrame#Main_QF{
	background-color: rgb(159, 246, 252);
border:0px solid red;
border-radius:30px
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="CameraIP">
         <property name="styleSheet">
          <string notr="true">QLineEdit{margin-left: 20px; margin-right: 20px;}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="4" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLineEdit" name="camera_ip">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>18</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{margin-left: 20px; margin-right: 20px; border: 1px solid rgb(0, 0, 0);}</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 700 italic 11pt &quot;Segoe UI&quot;;
margin-left: 10px;}</string>
            </property>
            <property name="text">
             <string>Local Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLineEdit" name="local_name">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>18</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{margin-left: 20px; margin-right: 20px; border: 1px solid rgb(0, 0, 0);}</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="top">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame#top{
background-color: rgba(255, 255, 255,0);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="explain_title">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>Register New Camera</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QFrame" name="buttons_sf">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="min_sf">
                  <property name="minimumSize">
                   <size>
                    <width>14</width>
                    <height>14</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>14</width>
                    <height>14</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	
	background-color: rgb(4, 180, 0);
border:1px solid rgba(113, 17, 15,50);
border-radius:6px;
}
QPushButton:hover {
background-color:rgb(62, 158, 47)
	
}
QPushButton:pressed {
	background-color: rgb(84, 171, 67);
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="max_sf">
                  <property name="minimumSize">
                   <size>
                    <width>14</width>
                    <height>14</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>14</width>
                    <height>14</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	
	background-color: rgb(227, 199, 0);
border:1px solid rgba(113, 17, 15,50);
border-radius:6px;
}
QPushButton:hover {
background-color:rgb(219, 209, 68)
	
}
QPushButton:pressed {
	background-color: rgb(209, 202, 96);
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="close_button">
                  <property name="minimumSize">
                   <size>
                    <width>14</width>
                    <height>14</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>14</width>
                    <height>14</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	
	background-color: rgb(240, 108, 96);
border:1px solid rgba(113, 17, 15,50);
border-radius:6px;
}
QPushButton:hover {
background-color:rgb(139, 29, 31)
	
}
QPushButton:pressed {
	background-color: rgb(232, 59, 35);
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="register_2">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	margin-top: 10px;
	margin-left: 20px;
	margin-right: 20px;
	margin-bottom: 5px;
	background-color: rgb(13, 63, 112);
	border-radius:6px;
	color: rgb(255,255,255);
	font-size: 12px;
}
QPushButton:hover {
background-color:rgb(45, 180, 204)
}
QPushButton:pressed {
	background-color: rgb(21, 140, 1617);
}</string>
            </property>
            <property name="text">
             <string>Register</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
font: 700 italic 11pt &quot;Segoe UI&quot;;
margin-left: 10px;}</string>
            </property>
            <property name="text">
             <string>Camera IP</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
